### ISSUE TITLE
Remnantsaveguardian is abandoned?

### ISSUE THREAD

**Issue by remnantstranger** [Razzmatazzz/RemnantSaveGuardian#242]
I've been in several months without receiving if you want an update, I think this project is the one that most helps the community of this game!
I would like to know if it was closed?, or will not receive updates anymore?
I would like to know something about, some possible update date with the new dlç.
Or should I give up?

**Comment by drkrthnblck** [comment]
> I've been in several months without receiving if you want an update, I think this project is the one that most helps the community of this game! I would like to know if it was closed?, or will not receive updates anymore? I would like to know something about, some possible update date with the new dlç. Or should I give up?

Nah, Nasbit has been working on it. Here's the [pre-release version](https://github.com/Razzmatazzz/RemnantSaveGuardian/pull/236#issuecomment-2091535748) he's been working on.

**Comment by Q-Sharp** [comment]
I don't know why but @Razzmatazzz seems to ignore pr for this project and remnant from the ashes pr's too. :(

**Comment by razeedazee** [comment]
Ask for a refund.

**Comment by Melabones** [comment]
Give them time to update it. The DLC just recently released and if it's just one person doing the updates that takes time especially when it's a free download and they don't have all there time to invest in the project. Also the analyzer still works at analyzing the DLC items. Rings and Amulets show as DLC2 and will tell you which dungeon they are located. Though sometimes it can get the location wrong but usually it's correct most of the time. Only thing is it doesn't tell what you're missing. You can easily keep track of which items you have already collected using like a notepad app and use the Wiki to find out all the new items for the DLC2 at least until the Analyzer is updated.

**Comment by Razzmatazzz** [comment]
I don't have as much free time as I used to, which means I don't have time to actively develop this project. I do have the bandwidth to merge PRs from anyone who wants to contribute and push out new releases.

@Q-Sharp apologies for taking so long to merge that RSM PR. It was opened around the holidays, I meant to get to it, and then I honestly forgot.

TheNasbit is working on a PR for the latest DLC in Remnant II, but it appears to be a work in progress with a couple items incomplete. Some folks in that PR also appear to be reporting regressions in how some events appear.
